target/linux/ath79/base-files/etc/board.d/02_network

```diff
@@ -260,6 +260,7 @@ ath79_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"1:lan" "2:lan" "3:lan" "4:lan" "0:wan" "9@eth0"
 		;;
+	tplink,tl-mr3420-v2|\
```

Comment:
This case should be moved above tplink,tl-wr1043nd-v1

Reply:
TL-MR3420 follows tl-wr841-v8 and tl-wr842n-v2 switch config NOT tl-wr1043nd-v1

Reply:
I mean this:
```diff
--- a/target/linux/ath79/base-files/etc/board.d/02_network
+++ b/target/linux/ath79/base-files/etc/board.d/02_network
@@ -248,6 +248,15 @@ ath79_setup_interfaces()
                ucidef_add_switch "switch0" \
                        "0@eth0" "2:lan:3" "3:lan:2" "4:lan:1" "1:wan"
                ;;
+       tplink,mt3420-v2|\
+       tplink,tl-wr740n-v4|\
+       tplink,tl-wr741nd-v4|\
+       tplink,tl-wr841-v8|\
+       tplink,tl-wr842n-v2)
+               ucidef_set_interface_wan "eth1"
+               ucidef_add_switch "switch0" \
+                       "0@eth0" "1:lan:4" "2:lan:1" "3:lan:2" "4:lan:3"
+               ;;
        tplink,tl-wr1043nd-v1)
                ucidef_add_switch "switch0" \
                        "1:lan" "2:lan" "3:lan" "4:lan" "0:wan" "5@eth0"
@@ -261,14 +270,6 @@ ath79_setup_interfaces()
                ucidef_add_switch "switch0" \
                        "1:lan" "2:lan" "3:lan" "4:lan" "0:wan" "9@eth0"
                ;;
-       tplink,tl-wr740n-v4|\
-       tplink,tl-wr741nd-v4|\
-       tplink,tl-wr841-v8|\
-       tplink,tl-wr842n-v2)
-               ucidef_set_interface_wan "eth1"
-               ucidef_add_switch "switch0" \
-                       "0@eth0" "1:lan:4" "2:lan:1" "3:lan:2" "4:lan:3"
-               ;;
        tplink,tl-wr941-v2)
                ucidef_set_interfaces_lan_wan "lan1 lan2 lan3 lan4" "wan"
                ;;
```

Same goes for the led change above.

Reply:
My change should only reflect the new hardware support.
If the change is to shift the entire block it should be a separate commit by someone who wants to rearrange the block
My commit should not change the sequence of the existing structure

Reply:
> My commit should not change the sequence of the existing structure

The existing structure is:
1. device names inside a block should be alphabetically ordered.
2. every case blocks should be alphabetically ordered according to the first device name.

Without this change, you are breaking the already maintained structure.

Reply:
Can you edit the blocks commits? If I were to do it I will need to do another pull request